bin/run_clang_tidy.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+BUILD_PATH=/build/faasm/
+CONFIG=${FAASM_ROOT}/.clang-tidy
+
+FILES=$(git ls-files {func,libs,src,tests}/{"*.h","*.cpp","*.c"})
+
+if [ -z "$1" ]; then
+    :
+else
+    pushd $1 >> /dev/null
+    FILES=$(git ls-files {"*.h","*.cpp","*.c"})
```

Comment:
Before, running the `git ls-files` on a directory different than root would give no results (as no `func`, `libs`, ... would match).